src/etl/veptranscript.py

```diff
@@ -165,18 +165,18 @@ def get_generators(self, filepath):
                               "impact": impact,
                               "gene": geneId,
                               "transcriptId": columns[4],
-                              "amino_acid_reference": amino_acid_reference,
-                              "amino_acid_variation": amino_acid_variation,
-                              "amino_acid_change": amino_acid_change,
-                              "cdna_start_position": cdna_start_position,
-                              "cdna_end_position": cdna_end_position,
-                              "cdna_range": cdna_range,
-                              "cds_start_position": cds_start_position,
-                              "cds_end_position": cds_end_position,
-                              "cds_range": cds_range,
-                              "protein_start_position":protein_start_position,
-                              "protein_end_position":protein_end_position,
-                              "protein_range": protein_range}
+                              "aminoAcidReference": amino_acid_reference,
```

Comment:
@sierra-moxon I am trying to clarify what you are talking about and am fully willing to change things to what we can agree on. In the code cleanup pull request, I am not changing the keys in theses sorts of maps to underscore. I have changed the variables including the ones on the right to underscore. I would have to go through all the changes made but I don't think i was changing any of the variables in the cypher queries either. In this way the variables that were changed should be caught as an error by python. If you have seen anything otherwise please let me know. If this is not what you mean please expand upon your thoughts.

Reply:
I guess what I am saying is that during development, this good python standard of underscored variable names (that for the Alliance standard, needs to be translated to camelCase in either a map or a cypher query string, two places that an IDE won't catch the subtle change)), might be error prone (my PR yesterday got caught up in this as I tried to comply with the underscored standard for python and the camelCase standard for Alliance).  I'm trying to think of constructive ways around this to decrease errors and comply with both standards.  Perhaps one way to get around these translation errors, is to have good test coverage.  An item for our development lists might be to investigate a tool like: https://coverage.readthedocs.io/en/coverage-5.1/ ?

Reply:
I agree. I was thinking it might be possible to make something if we have standard ways of doing this to have one test that scrapes through every ETL and looks for the map and then checks it with the queries. Makes sure the query is asking for keys that are in the map. etc. I would definitely help you with this.

Reply:
Coverage is fantastic, it would be great to use here. Coveralls.io provides provides this service for free (for open-source projects) with an excellent web UI.

Reply:
https://coveralls.io/